src/components/Api.js

```diff
@@ -0,0 +1,130 @@
+export default class Api {
+  constructor({ baseUrl, headers }) {
+    this._baseUrl = baseUrl;
+    this._auth = headers.authorization;
+    this._contentType = headers.contentType;
+  }
+
+  getUser() {
+    return fetch(`${this._baseUrl}/users/me`, {
+      headers: { authorization: this._auth },
```

Comment:
You should store the whole headers object in the constructor and reuse it in all the other methods

```js
headers: this._headers
```